AppCenterDataStorage/AppCenterDataStorage/MSDataStore.h

```diff
@@ -181,10 +181,10 @@ typedef void (^MSPaginatedDocumentsCompletionHandler)(MSPaginatedDocuments<T> *d
  * @param writeOptions Options for deleting the document.
  * @param completionHandler Callback to accept any errors.
  */
-+ (void)deleteDocumentWithPartition:(NSString *)partition
-                         documentId:(NSString *)documentId
-                       writeOptions:(MSWriteOptions *_Nullable)writeOptions
-                  completionHandler:(MSDocumentWrapperCompletionHandler)completionHandler;
++ (void)deleteWithPartition:(NSString *)partition
```

Comment:
Did we get approval to change the public API names? What is the purpose of this and how does it relate to this PR?

Reply:
Same as above.